502 error during the build for android

Recently, I can't build my project. The error is appearing referring to the repo is not available.

https://dl.bintray.com/intercom/intercom-maven/com/facebook/react/react-native/maven-metadata.xml is returning 504 status code.


Error:

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':react-native-intercom:generateReleaseRFile'.
> Could not resolve all task dependencies for configuration ':react-native-intercom:releaseRuntimeClasspath'.
   > Could not resolve com.facebook.react:react-native:+.
     Required by:
         project :react-native-intercom
      > Failed to list versions for com.facebook.react:react-native.
         > Unable to load Maven meta-data from https://dl.bintray.com/intercom/intercom-maven/com/facebook/react/react-native/maven-metadata.xml.
            > Could not get resource 'https://dl.bintray.com/intercom/intercom-maven/com/facebook/react/react-native/maven-metadata.xml'.
               > Could not GET 'https://dl.bintray.com/intercom/intercom-maven/com/facebook/react/react-native/maven-metadata.xml'. Received status code 502 from server: Bad Gateway
```


### Environment
npm version: 18
io.intercom.android:intercom-sdk-base:7.+

Tried different versions and the result is the same

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,6 +1,6 @@
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
@@ -30,9 +30,15 @@ android {
 }
 
 repositories {
+    mavenLocal()
     mavenCentral()
     maven {
-      url 'https://dl.bintray.com/intercom/intercom-maven'
+        // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
+        url "$rootDir/../node_modules/react-native/android"
+    }
+    maven {
+        // Android JSC is installed from npm
+        url "$rootDir/../node_modules/jsc-android/dist"
     }
 }
 

